integration/testing.py

```diff
@@ -34,6 +34,20 @@ def generate_object_name(self):
 
     def create_container(self):
         """Create a container in lxd."""
+        if not getattr(type(self), '_busybox_ready', False):
+            path, fingerprint = create_busybox_image()
+
+            with open(path) as f:
+                self.client.images.create(f.read(), wait=True)
+
+            self.lxd.images.aliases.post(json={
+                'description': '',
+                'target': fingerprint,
+                'name': 'busybox',
+            })
+
+            type(self)._busybox_ready = True
+
```

Comment:
See line 59 of this file how it expects the 'busybox' image to be there.
![2016-05-24-204543_537x198_scrot](https://cloud.githubusercontent.com/assets/94636/15515966/d357a896-21f0-11e6-8f52-7c21fc77a6de.png)